### PR TITLE
net-vpn/logmein-hamachi: version bump, bug #696538

### PR DESCRIPTION
**Pull request by nonno-cicala**
The license was updated so I renamed the old, unversioned, license to allow versioning.

I've been using version 2.1.0.198 for a long time and I consider it stable for amd64.

### CODE REVIEW

File: net-vpn/logmein-hamachi/logmein-hamachi-2.1.0.198.ebuild [hunk 1]
```diff
@@ -9,9 +9,9 @@ HOMEPAGE="https://www.vpn.net/"
 SRC_URI="x86?	( https://www.vpn.net/installers/${P}-x86.tgz )
 	amd64?	( https://www.vpn.net/installers/${P}-x64.tgz )"
 
-LICENSE="LogMeIn"
+LICENSE="LogMeIn-2017"
 SLOT="0"
-KEYWORDS="-* ~amd64 ~x86"
+KEYWORDS="-* amd64 ~x86"
```
Comment (juippis) [inline net-vpn/logmein-hamachi/logmein-hamachi-2.1.0.198.ebuild]: You can not perform this action. Only members of arch team can do the stabilization. Just focus on renaming the license, and request a stabilization for this version through bugzilla.

File: net-vpn/logmein-hamachi/logmein-hamachi-2.1.0.203.ebuild [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (juippis) [inline net-vpn/logmein-hamachi/logmein-hamachi-2.1.0.203.ebuild]: I wonder if 7 works here, namely `linux-info` should support it, but I havent tested it yet. 
Also please one empty line below EAPI to follow default ebuild skel format.

File: net-vpn/logmein-hamachi/logmein-hamachi-2.1.0.198.ebuild (hunk 1 above)
Comment (nonno-cicala) [reply net-vpn/logmein-hamachi/logmein-hamachi-2.1.0.198.ebuild]: I reverted the stabilization part, thanks

File: net-vpn/logmein-hamachi/logmein-hamachi-2.1.0.203.ebuild (hunk 2 above)
Comment (nonno-cicala) [reply net-vpn/logmein-hamachi/logmein-hamachi-2.1.0.203.ebuild]: I tried 7 and it seems to work fine, thanks

**Comment by nonno-cicala** [comment]
I signed-off the commits and I hope they are short enough now.
Is there something else I should do?
Thanks 🙂

**Comment by nonno-cicala** [comment]
> Then ,you need to sign-off ALL your commits, and shorten the 2nd one (you can remove the stabilization part so it should become short enough for summary).

I signed-off the commits and I hope they are short enough now.
Is there something else I should do?
Thanks 🙂

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**